Source/UIKit/UIActivityIndicatorView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UIActivityIndicatorView.swift
+//  Rex
+//
+//  Created by Evgeny Kazakov on 02/07/16.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import ReactiveCocoa
+import UIKit
+
+extension UIActivityIndicatorView {
+
+    /// Returns whether the receiver is animating.
+    /// `true` if the receiver is animating, otherwise `false`.
+    ///
+    /// Setting the value of this property to `true` starts animation of the progress indicator,
+    /// and setting it to `false` stops animation.
+    public var animating: Bool {
```

Comment:
Sorry for the late, but this seems confusing to me. I think the logic should be implemented in `rex_animating` directly.